app/controllers/sessions_controller.rb

```diff
@@ -22,7 +23,7 @@ def create
       redirect_to user_redirect_path
 
     else
-      redirect_to root_path, alert: I18n.t("sessions.create.login_error")
+      redirect_to root_path(login_failed: true)
```

Comment:
NIT: not a huge fan of using flags like this in the params. Any reason to not do the following?
1) use `render_to_string` to render the login error message list into the flash
2) move/restyle the flash in the "public" layout to be in a more user friendly place